--- a/package/lean/autocore/files/x86/autocore
+++ b/package/lean/autocore/files/x86/autocore
@@ -28,11 +28,10 @@ start()
 	d=$(echo -n ' Core ')
 	e=$(cat /proc/cpuinfo | grep 'processor' | wc -l)
 	f=$(echo -n ' Thread ')
-	g=$(cat /tmp/sysinfo/model)
-	h=${g}' - '${a}${b}${c}${d}${e}${f}
+	g=${a}${b}${c}${d}${e}${f}
 
 	mkdir -p /tmp/sysinfo
-	echo $h > /tmp/sysinfo/model
+	echo $g > /tmp/sysinfo/model
 
 	a=$(ip address | grep ^[0-9] | awk -F: '{print $2}' | sed "s/ //g" | grep '^[e]' | grep -v "@" | grep -v "\.")
 	b=$(echo "$a" | wc -l)
